### DISCUSSION TITLE
Option to specify the location folders are backed up to

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#892] [Enhancements]
**Describe the feature**
A way to specify where each backup folders goes would be useful for customers who would prefer a clean interface without too many albums.
For example, they might want a single album that contains all the photos from Whatsapp, Signal, etc.

**Comment by hadrienj** [comment]
Totally agree @vishnukvmd. As for me, I don't know how to separate backup from two Android phones with the folder named 'Camera', the photos from both phones go into the same Ente folder which is annoying.
